Makefile

```
@@ -57,7 +57,7 @@ extension: build/unload-client.js
 extension: build/pdfjs-init.js
 extension: $(addprefix build/,$(EXTENSION_SRC))
 
-build/extension.bundle.js: src/background/*.js rollup.config.mjs build/settings.json
+build/extension.bundle.js: src/background/*.js src/background/*.ts rollup.config.mjs build/settings.json
```

If this is a glob pattern, maybe we could "extend" the existing path to support both extensions.

```suggestion
build/extension.bundle.js: src/background/*.{js,ts} rollup.config.mjs build/settings.json
```

---

This glob syntax isn't supported by Make. I'm not aware of a direct equivalent, though `src/background/*` probably would work fine here.

---

> This glob syntax isn't supported by Make

That's too bad! 😅